cmd/adm_graphql.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func PrintAffiliationsGraphql(cmd *cobra.Command, args []string) {
+
+	configNamesGraphqlRequest := `{auroraApiMetadata{configNames}}`
+	type ConfigNamesResponse struct {
+		AuroraApiMetadata struct {
+			ConfigNames []string
+		}
+	}
+
+	var configNamesResponse ConfigNamesResponse
+	if err := DefaultApiClient.RunGraphQl(configNamesGraphqlRequest, &configNamesResponse); err != nil {
+		return
```

Comment:
Hva skjer hvis man feiler her? Hva blir printet ut da? Får vi noen feilmelding ut da?

Reply:
Man får feilene som produseres av github.com/machinebox/graphql/Client.Run direkte.  Jeg har sett at man får feilmeldinger (f.eks. dersom man sender inn en graphql-request som ikke er støttet av gobo), men har ikke gjort systematiske feilkall for å fremprovosere ulike typer feil.

Reply:
Jeg har endret dette slik at man får sendt err videre slik at det håndteres og skrives ut av cobra. Merk at dette ikke var slik for ao adm affiliations for booberkall tidligere. I det gamle kallet ble også err "svelget".  Om det skulle vise seg å være kryptiske meldinger, så får vi sørge for å fange slike meldinger og gi bedre tilbakemelding. Jeg synes det er bedre å gi kryptisk informasjon om at noe faktisk gikk galt enn å ikke gi noen informasjon. Det gir bedre utgangspunkt for vår debugging.